lab/lab2/js/part3-application.js

```diff
@@ -39,17 +40,26 @@ var resetMap = function() {
   it down!
 ===================== */
 var getAndParseData = function() {
-  /* =====================
-    Fill out this function definition
-  ===================== */
+  $.ajax(phillySolarInstallationDataUrl).done(function(data){ myData = JSON.parse(data); });
```

Comment:
can you access myData outside of the ajax wrapper? this doesn't seem to work when I do it

Reply:
I had to redefine phillySolarInstallationDataUrl in this file, but I was able to access myData as a global variable.